packages/cyberstorm/src/index.ts

```diff
@@ -84,6 +84,7 @@ export {
 export { Title, type TitleProps } from "./components/Title/Title";
 export { Tooltip, type TooltipProps } from "./components/Tooltip/Tooltip";
 export { Alert, type AlertProps } from "./components/Alert/Alert";
+export { Toast, type ToastProps } from "./components/Toast/Toast";
```

Comment:
As mentioned in the other comment's I would refrain from exporting the interface outside the package if it isn't absolutely needed.

Reply:
You've reacted with 👍🏻 but `ToastProps` is still being exported. Pointing this out to make sure the decision is intentional

Reply:
Yep was intentional, since it's needed in the story

Reply:
It's not needed in story. Do `variant: "info" as const` which tells TS that the type is "info" and not a string and you're good to go without imports.

Reply:
Huh, didn't think of that. Nice.